src/olympia/devhub/tasks.py

```diff
@@ -73,6 +73,7 @@ def validate(file_, *, final_task=None, theme_specific=False):
     if task_id:
         return AsyncResult(task_id)
     else:
+        breakpoint()
```

Comment:
why?

Reply:
I since removed that breakpoint (was only there while I was debugging)